packages/rendering/browser.js

```diff
@@ -19,6 +19,8 @@ if (module.hot) {
 const container = document.getElementById('app');
 
 if (container) {
-    hydrateCSS(cssIDs);
+    if (process.env.NODE_ENV !== 'development') {
```

Comment:
I have to prevent emotion hydrating styles in the browser in `development` mode to retain the sourceMap info. As detailed in the issue raised here https://github.com/emotion-js/emotion/issues/487

Reply:
Shall we add a TODO or another comment here to remind us that this issue exists?

Reply:
Yep good suggestion!